app/build.gradle

```diff
@@ -98,7 +98,7 @@ dependencies {
     implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.2.1'
     implementation 'androidx.browser:browser:1.0.0'
     implementation 'androidx.lifecycle:lifecycle-extensions:2.0.0'
-    implementation group: 'com.google.android.play', name: 'core', version: '1.6.0'
+    implementation 'com.google.android.play:core:1.6.0'
```

Comment:
@aggarwalpulkit596 Let me know if you agree with this. I shall remove it accordingly